package.json

```diff
@@ -15,7 +15,8 @@
     "build:dev": "webpack --mode=development && npm run build:es",
     "docs:examples": "node tooling/docs/examples-to-md.js examples/node/*.js",
     "docs:api": "node apidoc.js",
-    "lint": "commitlint --from HEAD~50 && eslint .",
+    "docs:changelog": "cp CHANGELOG.md docs/CHANGELOG.md",
```

Comment:
why this again? I moved the changelog file in the release PR for reasons of linking to correct sources in repo and mkdocs documentation

Reply:
was checking it.
wait undoing this.